accounts/views.py

```diff
@@ -23,3 +28,20 @@ def get_permissions(self):
 
 
 
+class ObtainAuthToken(APIView):
+    throttle_classes = ()
+    permission_classes = ()
+    parser_classes = (parsers.FormParser, parsers.MultiPartParser, parsers.JSONParser,)
+    renderer_classes = (renderers.JSONRenderer,)
+    serializer_class = AuthTokenSerializer
+
+    def post(self, request, *args, **kwargs):
+        serializer = self.serializer_class(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        user = serializer.validated_data['user']
+        Token.objects.filter(user=user).delete()
+        token = Token.objects.create(user=user)
+        return Response({'token': token.key})
+
+
+obtain_auth_token = ObtainAuthToken.as_view()
```

Comment:
I'm not sure so just asking..is it better to put this here or put under urls.py (.as_view()). Because it is a bit strange to have a variable outside a valid function in the bottom of the code like this. Alternatively, is it possible to change the type of the function so this is not needed?